src/State/Scope.php

```diff
@@ -303,10 +303,9 @@ public function clear(): self
      * Applies the current context and fingerprint to the event. If the event has
      * already some breadcrumbs on it, the ones from this scope won't get merged.
      *
-     * @param Event                      $event   The event object that will be enriched with scope data
-     * @param array<string, mixed>|Event $payload The raw payload of the event that will be propagated to the event processors
+     * @param Event $event The event object that will be enriched with scope data
      */
-    public function applyToEvent(Event $event, $payload): ?Event
```

Comment:
See one of the comment above about why this argument should not be dropped, but it should became the `$hint` instead